msmbuilder/featurizer/featurizer.py

```diff
@@ -997,6 +1008,36 @@ def partial_transform(self, traj):
         return md.shrake_rupley(traj, mode=self.mode, **self.kwargs)
 
 
+class DSSPFeaturizer(Featurizer):
+    """Featurizer based on Dictionary of protein secondary structure
```

Comment:
The class docstring needs to be just one line for command line compatibility. You can then add line breaks and explain more. See the other featurizers for examples.